.devops/pagopa-code-review-pipelines.yml

```diff
@@ -122,3 +130,70 @@ stages:
           inputs:
             codeCoverageTool: Cobertura
             summaryFileLocation: 'coverage/cobertura-coverage.xml'      
+
+  # B) Run integration tests if there is a push or pull request on any branch.
+  - stage: Integration_Tests
+    pool:
+      vmImage: 'ubuntu-latest'
+    dependsOn: []
+    jobs:
+      - job: run_integration_tests
+        timeoutInMinutes: 30
+        steps:
+
+        - checkout: wallet-be-mock
+
+        # Currently npg api key is not required for current integration test
+        - script: |
+            cp env.example .env
+            sed -i "s/^NPG_API_KEY=.*$/NPG_API_KEY=key/" .env
+          workingDirectory: pagopa-wallet-be-mock
+          displayName: 'Setup env for mock'
+
+        - script: yarn global add node-gyp && yarn install && yarn generate && yarn build
+          workingDirectory: pagopa-wallet-be-mock
+          displayName: 'Setup deps for mock'
+
+        - script: |
+            nohup yarn start &
+          displayName: 'Start backend mock'
+          workingDirectory: pagopa-wallet-be-mock
+
+        - checkout: self
+
+        - template: templates/node-job-setup/template.yaml@pagopaCommons
+          parameters:
+           projectDir: pagopa-wallet-fe
+
+        - script: |
+            cp env.example .env.development
+          displayName: 'copy env.example to .env.development'
+          workingDirectory: pagopa-wallet-fe
+
+        - script: |
+            yarn dev &
```

Comment:
Why do you start this? And then do you start it with `dev:noredirect`? I don't see any integration tests after this startup.